external/Makefile.tools

```diff
@@ -48,6 +48,7 @@ gnu_build:
 	cd $(gnu_dir); \
 		./configure --prefix=$(BP_EXTERNAL_DIR) --with-arch=rv64ima --with-abi=lp64 --with-cmodel=medany 
 	$(MAKE) -C $(gnu_dir)
+	$(MAKE) -C $(gnu_dir) linux
```

Comment:
Does this definitely generate both toolchains?

Reply:
Yeah I tested it from scratch.